website/react-magma-docs/src/pages/api/popover.mdx

```diff
@@ -0,0 +1,467 @@
+          Disabled Popover
+        </PopoverTrigger>
+        <PopoverContent>
+          <div>Popover is disabled</div>
+        </PopoverContent>
+      </Popover>
+      <Popover hoverable>
+        <PopoverTrigger aria-label="Open popover">
+          <Button disabled>Disabled Trigger</Button>
+        </PopoverTrigger>
+        <PopoverContent>
+          <div>Popover is enabled, trigger is disabled</div>
+        </PopoverContent>
+      </Popover>
+    </div>
+  );
+}
+```
+
+## Open by default
+
+The `openByDefault` property will allow Popover to open as soon as the page loads. It is recommended to use localStorage to modify this behavior.
+
+```tsx
+import React from 'react';
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  Button,
+  FormGroup,
+  Checkbox,
+  ButtonSize,
+} from 'react-magma-dom';
+
+export function Example() {
+  const popoverApiRef = React.useRef<PopoverApi>();
+
+  function handleClose(event: React.SyntheticEvent) {
+    if (popoverApiRef.current) {
+      popoverApiRef.current.closePopoverManually(event);
+    }
+  }
+
+  const sessionStorageValue = JSON.parse(
+    sessionStorage.getItem('temporaryOpenByDefault')!
+  );
+
+  const initialOpen = React.useMemo(() => {
+    return sessionStorageValue || false;
+  }, []);
+
+  const [checked, setChecked] = React.useState(() => initialOpen);
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const nextValue = !event.target.checked;
+
+    sessionStorage.setItem('temporaryOpenByDefault', JSON.stringify(nextValue));
+
+    setChecked(nextValue);
+  };
+
+  return (
+    <Popover openByDefault={initialOpen} apiRef={popoverApiRef} width={320}>
+      <PopoverTrigger />
+      <PopoverContent>
+        <span>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt.
+        </span>
+        <FormGroup
+          containerStyle={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+          }}
+        >
+          <Checkbox
+            labelText="Don’t show this again"
+            onChange={onChange}
+            checked={!checked}
+          />
+          <Button size={ButtonSize.small} onClick={handleClose}>
+            Got it
+          </Button>
+        </FormGroup>
+      </PopoverContent>
+    </Popover>
+  );
+}
+```
+
+## Popover Props
+
+**Any additional props supplied will be provided to the wrapping div element.**
+
+<PopoverProps />
+
+## Popover Content Props
+
+**Props supplied will be passed to the wrapping div element containing the popover content.**
+
+<PopoverContentProps />
+
+## Popover Section Props
+
+**Any additional props supplied will be provided to the wrapping section element.**
+
+<PopoverSectionProps />
+
+## Popover Trigger Props
+
+**Props supplied to the PopoverTrigger component will be provided to the underlying button element.**
+
+<PopoverTriggerProps />
```

Comment:
For some reason, these are not visible in the docs page
![image](https://github.com/user-attachments/assets/7825f508-796c-418c-9f8e-feeae2729244)